src/signer.rs

```diff
@@ -6,10 +6,14 @@ use rustls::{
     sign::{Signer, SigningKey},
     Error, OtherError, SignatureAlgorithm, SignatureScheme,
 };
-use sha2::digest::Digest;
```

Comment:
Do you need to remove sha2 from Cargo.toml?

Reply:
What about the aws-lc-rs dependency in Cargo.toml

Reply:
done